### PR TITLE
pom.xml was modified

### PR DESCRIPTION
**Pull request by git-user-cpp**

### CODE REVIEW

File: MainModule/src/main/java/model/data/TableHolder.java [hunk 1]
```diff
@@ -15,25 +15,59 @@ public class TableHolder {
...
-    public void addRow(HashMap<String, String> rowDataPackage) {
+    public String addRow(HashMap<String, String> rowDataPackage) {
         if (!verifyColumnNamesCorrect(rowDataPackage)) {
-            //TODO Create respectful exception schema
-            throw new RuntimeException("Incorrect rows in insert data package");
+            throw new IllegalArgumentException("[ERROR] Incorrect rows in insert data package");
         }
         rows.put(generateUniqueId(), rowDataPackage);
+
+        return returnUniqueId();
```
Comment (VladimirPetrat) [inline MainModule/src/main/java/model/data/TableHolder.java]: Bug

File: MainModule/src/main/java/model/data/TableHolder.java [hunk 2]
```diff
@@ -15,25 +15,59 @@ public class TableHolder {
...
+    }
+
+    public void removeRow(String id) throws IllegalAccessException {
```
Comment (VladimirPetrat) [inline MainModule/src/main/java/model/data/TableHolder.java]: Why method throws exception up?

File: MainModule/src/main/java/model/data/Table.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        return rows
+                .get(Id)
+                .stream()
+                .filter(dataStructure -> dataStructure.getFieldName().equals(fieldName))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException("Missing field name"))
+                .getValue();
+    }
+
+    public DataObject getDataForFieldName(String Id, String fieldName) {
```
Comment (VladimirPetrat) [inline MainModule/src/main/java/model/data/Table.java]: Same method as previous. Re-do or drop.

Comment (git-user-cpp) [reply MainModule/src/main/java/model/data/Table.java]: My bad) Done

**Review by VladimirPetrat — APPROVED** [review]

**Pull request merged**